libs/design/src/atoms/button/button.component.scss

```diff
@@ -1,11 +1,11 @@
 @import '../../scss/daff-util';
 
-@mixin daff-button {
+@mixin button-styles {
     display: inline-block;
     position: relative;
     font-family: $body-font-family;
-    font-size: 1rem;
-    margin: 0;
+    font-size: $medium-font-size;
+    border-radius: 3px;
```

Comment:
This has unintended side-effects, see the button in the footer of `daffio`.

Reply:
I was going to wait for the Input PR to get merged before updating the styles for newsletter form.

Reply:
@xelaint in that case, I'll rename this to WIP and mark this out-of-scope for 0.2.

Reply:
@damienwebdev updated the newsletter styles